cloudbridge/cloud/providers/openstack/services.py

```diff
@@ -210,14 +210,14 @@ def list(self, limit=None, marker=None):
         return ClientPagedResultList(self.provider, firewalls,
                                      limit=limit, marker=marker)
 
-    def create(self, label, description, network_id):
+    @cb_helpers.deprecated_alias(network_id='network')
```

Comment:
@almahmoud I think the original order is more intuitive when reading the code? 
```suggestion
    @cb_helpers.deprecated_alias(network='network_id')
```

Reply:
It didn't work properly with that order though since it was trying to convert network to network_id, and that parameter wasn't recognized. We could potentially switch it by flipping things in the helper function, but it seems like a lot of work for just aesthetics imo.

Reply:
I agree it's not worth extra effort, especially given it'll be removed one day. Meanwhile, reading `@cb_helpers.deprecated_alias(network_id='network')` seems to me like the deprecated alias is `network_id` and it should be matched to `network`, no?